src/shogun/classifier/LDA.cpp

```diff
@@ -150,12 +140,14 @@ template <typename ST>
 bool CLDA::solver_classic()
 {
 	auto dense_feat = static_cast<CDenseFeatures<ST>*>(features);
-	index_t num_feat = dense_feat->get_num_features();
+  auto labels = multiclass_labels(m_labels);
```

Comment:
Have a look at the output of the style checker ;-)

Reply:
actually this pr needs some more work so i am holding style check off till the end :)

Reply:
For sure, then I won't be shouting about them 🙊